settings.gradle.kts

```diff
@@ -25,7 +25,8 @@ include(
 android.include(
         ":framework:android:kodein-di-framework-android-core",
         ":framework:android:kodein-di-framework-android-support",
-        ":framework:android:kodein-di-framework-android-x"
+        ":framework:android:kodein-di-framework-android-x",
+        ":framework:android:kodein-di-framework-android-x-viewmodel-savedstate"
```

Comment:
I would suggest to rename the module without the `savedstate` mention, like in the manifest `android.x.viewmodel`.

Reply:
This has an impact on the transitive dependencies that this module would bring into the dependency graph. Katana had separated their basic ViewModel module from their ViewModel with SavedStateHandle module seemingly for this reason. Google packages ViewModel separately from the SavedStateHandle extensions, and apps may well make use of ViewModel without SavedStateHandle extensions. I assumed that Kodein DI would similarly try to minimize the inclusion of potentially unwanted transitive dependencies.

I don't have any too strong feelings about this one way or the other. Proguard/R8 could strip out truly unused components. I would be happy to combine into a single module if that is indeed your preference. Just wanted to raise the potential concern here.

Reply:
Ok then, let's go for the module `:framework:android:kodein-di-framework-android-x-viewmodel-savedstate` to follow Google practices.

So, for the plain ViewModel you will go with another module `:framework:android:kodein-di-framework-android-x-viewmodel` ?

Reply:
Sounds good. I will do separate modules with those names.